src/views/Viewer.vue

```diff
@@ -1280,7 +1280,8 @@ export default {
 			background-color: rgba(255, 255, 255, .08) !important;
 		}
 		&:focus,
-		&:focus-visible {
+		&:focus-visible,
+		&.action-item__menutoggle {
```

Comment:
`.action-item__menutoggle` should have its own declaration instead of sharing it with `:focus` and `:focus-visible` as we don't want to have the permanent outline